CMakeLists.txt

```diff
@@ -504,10 +504,13 @@ endif()
 find_package(BISON 3.0.4 REQUIRED)
 find_package(FLEX 2.5.13 REQUIRED)
 
+if(${VELOX_ENABLE_DUCKDB})
+  set_source(DuckDB)
```

Comment:
Do we want to set the minimum version for system duckdb? Or is that not critical?

Reply:
It is safer to have a minimum version so that tests are compared consistently against newer versions.
But DuckDB does not set the right version when building from tarball
https://github.com/duckdb/duckdb/blob/44d7f80c3e5cb3e4e9f57814e74a9baac13c8fbf/CMakeLists.txt#L280

Reply:
Can we build from a git tag?

Reply:
git tag won't solve the problem since a system install can still come from a tarball.

Reply:
Oh they hardcode 0.0.1 for tarball builds... weird choice :/  but nothing we can do about it... (we could fallback to looking for 0.0.1 if we find no other version and accept anyway but that would require a bunch of changes that I don't think are worth it)